.github/actions/publish/action.yml

```diff
@@ -15,14 +15,19 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Checkout
-      uses: actions/checkout@v4
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Setup Docker Buildx
+      uses: docker/setup-buildx-action@v3
       with:
-        fetch-depth: 0
-    - name: Set up Go
-      uses: actions/setup-go@v5
-      with:
-        go-version: stable
```

Comment:
Looking at ld-relay. I'm fairly confident we don't need either of these steps

Reply:
If the calling workflow has a checkout step, you should be able to omit it here. It might even be useless in the Relay flow. 

The setup Go is because we run unit tests + build Relay before publishing